collections/videos.js

```diff
@@ -0,0 +1,63 @@
+//Collection:
+//Provide base URL with API key
+//View will provide the searchQuery
+//Model will listen and append the id
```

Comment:
I'm not sure what you mean by this... I'm not sure it's needed either.  Once the collection gets data, it should just re-render.